### ISSUE TITLE
YouTube videos recently stopped playing

### ISSUE THREAD

**Issue by jamminjames** [RemcoB00/cordova-phonegap_youtube_player_api_android#11]
I know this plugin is very old, but it was working fine until very recently. I created a fork to use in a Phonegap Build app, and included the plugin via the config.xml file with this line:
`<plugin name="remcob00-cordova-youtube-android-player-api" spec="jamminjames/cordova-phonegap_youtube_player_api_android" />`
Can anyone tell me why it would stop working? Is it a Youtube API update or something? Can it be dealt with in my fork of the plugin? Thanks for any help!

**Comment by jamminjames** [comment]
Update: I followed the link in the install.js file for the Youtube API:
https://developers.google.com/youtube/android/player/downloads/YouTubeAndroidPlayerApi-1.2.1.zip
... and the zip file is not there, it's been updated to YouTubeAndroidPlayerApi-1.2.2.zip. So I edited  install.js to reflect that. I also replaced the YouTubeAndroidPlayerApi.jar file in /src/android/ with the new .jar file that is in YouTubeAndroidPlayerApi-1.2.2.zip. 
However, it's still not working. Any help would be most appreciated.
